microsoft/PowerToys
# [KeyboardManager] Added option for exact match shortcut

## Summary of the Pull Request
Added option for exact match for a shortcut. If exact match is checked then the shortcut will only be replaced if no other keys are also pressed. 


## PR Checklist

- [X] **Closes:** #10961


## Detailed Description of the Pull Request / Additional comments
The exact match key is in the select keys window below the Allow Chords checkbox.
I also removed ShortcutSecondKeyOfChordSettingName from begin saved and loaded in json since its redundant since secondKey is already set in SetKeyCodes in constructor.




## Validation Steps Performed
1. Open Keyboard Manager
2. Create a shortcut and set it to exact match
3. Check that the shortcut only gets triggered if no other keys are pressed.
